src/RuleMatchedLog.php

```diff
@@ -261,31 +230,55 @@ public function showItemForm($items_id, $itemtype)
             'START'  => (int)$start,
             'LIMIT'  => (int)$_SESSION['glpilist_limit']
         ];
+        $rows = [];
         foreach ($DB->request($params) as $data) {
-            echo "<tr class='tab_bg_1'>";
-            echo "<td>";
-            echo Html::convDateTime($data['date']);
-            echo "</td>";
-            echo "<td>";
+            $row = [];
+            $row['date'] = Html::convDateTime($data['date']);
```

Comment:
Formating should be done on template side, using the `|formatted_datetime` filter.